### PR TITLE
set up puppet so launch configs work, including using singular name for ...

### PR DESCRIPTION
**Pull request by rhelmer**
...roles consistently

r? @phrawzty - I've only tested collector so far, which seems to WFM with these changes.

### CODE REVIEW

File: puppet/modules/socorro/manifests/role/admin.pp [hunk 1]
```diff
@@ -6,4 +6,12 @@
       ensure=> latest
   }
 
+  file {
+    '/etc/cron.d/socorro':
+      mode => 600,
+      owner => root,
+      group => root,
+      source => 'puppet:///modules/socorro/files/etc_cron.d/socorro'
```
Comment (rhelmer) [inline puppet/modules/socorro/manifests/role/admin.pp]: Will this work ok with masterless puppet? Not sure how to specify the path here if not (obv. I haven't tested this role either :) )

**Comment by rhelmer** [comment]
@phrawzty OK I read up on it a bit and fixed up the admin role. I went ahead and tested all socorro-specific roles (everything but ES, PG, Consul) and this seems to work fine for all of them, with the latest socorro RPM in the repo (which is build from my ```centos7-fpm-update``` branch) \o/

**Comment by phrawzty** [comment]
Looks good.  There are just some style nits to fix, though: lines 140-144 of the [Travis run](https://travis-ci.org/mozilla/socorro-infra/builds/55336749) are `puppet-lint` warnings.  Please to be fixing those before merge. :page_facing_up: Otherwise `r+`.

**Comment by phrawzty** [comment]
Also: suggest adding `--fail-on-warnings` as an arg to `puppet-lint` in `.travis.yml`.

**Pull request merged**